### PR TITLE
Testing ng

### LINKED ISSUE #28: Test user interaction with Form elements
Examples:
+ clicking `next`
+ clicking `submit`
+ choosing answers to the series of questions

### LINKED ISSUE #67: Allow user to see top of form after clicking next button

### PR DESCRIPTION
**Pull request by nicolegooden**
#### What's this PR do?  

+ Allows the user to automatically see the top of each question set within `Form`, rather than clicking the next button and needing to scroll up to see the first question
+ Removes initial test for App 
+ Adds unit, integration, and sad path testing for `Form`

#### Where should the reviewer start? 

Most changes have occurred in `Form.test.js` so start there to see the 4 tests I've created.  As of now, they are all passing.

#### How should this be manually tested?  

Run `npm test` to verify that the tests from `Form.test.js` are passing as expected.  Also, view the form component in the browser and ensure that you are taken to the top of the window when you click the next button.

#### Any background context you want to provide?  

Please let me know if my `Form` tests need to be more thorough in any way.

#### What are the relevant tickets?  

Closes #28  
Closes #67

### CODE REVIEW

File: src/Components/Form/Form.test.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    expect(screen.getByText('How old is too old?')).toBeInTheDocument();
+    userEvent.click(screen.getByText('Pop'));
+    userEvent.click(screen.getByText('Country'));
+    userEvent.click(screen.getByText('1990\'s'));
+    expect(mockUpdateActivityAnswers).toHaveBeenCalledTimes(5);
+    userEvent.click(screen.getByRole('button', {name: 'next'}));
+    const gameQuestion = await waitFor(() => screen.getByText('How many people are playing?'));
+    expect(gameQuestion).toBeInTheDocument();
+    expect(mockSetHistory).toHaveBeenCalledTimes(2);
+    expect(screen.getByRole('button', {name: 'submit'})).toBeInTheDocument();
```
Comment (BlakeDonn) [inline src/Components/Form/Form.test.js:97]: These tests look very thorough, definitely checks most edge cases for our form

File: src/Components/Form/Form.test.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    userEvent.click(screen.getByText('music'));
+    userEvent.click(screen.getByText('card games'));
+    userEvent.click(screen.getByRole('button', {name: 'next'}));
+    const nextQuestion = await waitFor(() => screen.getByText('Which music genre(s)?'))
+    expect(nextQuestion).toBeInTheDocument();
+    expect(mockSetHistory).toHaveBeenCalledTimes(1);
+    userEvent.click(screen.getByRole('button', {name: 'back'}));
+    const prevQuestion = await waitFor(() => screen.getByText('Which activities excite you right now?'));
+    expect(prevQuestion).toBeInTheDocument();
+    expect(mockGoBack).toHaveBeenCalledTimes(1);
```
Comment (BlakeDonn) [inline src/Components/Form/Form.test.js:168]: Good call integrating testing for the back button, glad to see it wasn't too much of a pain to do.

**Review by BlakeDonn — APPROVED** [review]
Very thorough testing overall, this was definitely our most intricate component, and I can't think of anything missing here!

File: src/Components/Form/Form.test.js (hunk 2 above)
Comment (nicolegooden) [reply src/Components/Form/Form.test.js:168]: Of course!

File: src/Components/Form/Form.js [hunk 3]
```diff
@@ -66,6 +66,7 @@ export class Form extends Component {
 
   updateAllAnswers = async (event) => {
     event.preventDefault();
+    window.scrollTo(0,0);
```
Comment (Atos20) [inline src/Components/Form/Form.js:69]: such a neat trick! thanks for the hard work you put into this.

File: src/Components/Form/Form.test.js (hunk 1 above)
Comment (Atos20) [reply src/Components/Form/Form.test.js:97]: Love the testing, thanks !

**Review by Atos20 — COMMENTED** [review]
Such a thorough test!

**Review by Atos20 — APPROVED** [review]
